spec/helpers/visit_helper_spec.rb

```diff
@@ -84,7 +84,9 @@ def visit
     end
 
     it "provides a formatted date for when a response may be sent out" do
-      expect(helper.when_to_expect_reply(Date.parse("2014-10-03"))).to eq("Monday  6 October")
+      Timecop.travel(Date.parse("2014-10-03")) do
+        expect(helper.when_to_expect_reply).to eq("Friday  3 October")
```

Comment:
Something like `d.strftime("%A") + ' ' + d.strftime("%e %B").strip` on [ApplicationHelper](https://github.com/ministryofjustice/prison-visits/blob/fix/prison-schedules-and-prison-working-days/app/helpers/application_helper.rb#L14), but more elegant.